mbofui/app/js/utils.js

```diff
@@ -43,7 +46,7 @@ var leafletize = function(data){
     return leafletList;
 };
 
-
+// placeholder
```

Comment:
An Angular template will be providing all the content of the popup - for now this will do.